modules/home-manager/gtk.nix

```
@@ -121,10 +123,17 @@ in
     (mkIf enable {
       gtk.theme =
         let
-          gtkTweaks = concatStringsSep "," cfg.tweaks;
+          gtkTweaks =
+            if cfg.tweaks == [ ] then
+              ""
+            else
+              (concatStrings [
+                "+"
+                (concatStringsSep "," cfg.tweaks)
+              ]);
```

```suggestion
          gtkTweaks = "+" + concatStringsSep "," cfg.tweaks;
```
there are like a thousand different ways you could do this, but i just found this to be a bit cleaner

---

Indeed, that's much better.